src/components/UserPanel/UpdateUserModal/UpdateUserModal.jsx

```diff
@@ -0,0 +1,509 @@
+              label="Your e-mail"
+              value={formik.values.email}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              error={formik.touched.email && Boolean(formik.errors.email)}
+              helperText={formik.touched.email && formik.errors.email}
+              variant="outlined"
+              color="grey"
+              className={css.email}
+              style={{ width: '315px', display: 'flex' }}
+              sx={{
+                border: 'none',
+                borderColor: 'grey.400',
+                paddingTop: '0px',
+                paddingBottom: '0px',
+                marginTop: '20px',
+                marginBottom: '0px',
+                height: '60px',
+
+                fieldset: {
+                  borderRadius: 0,
+                  border: 'none',
+                  width: '315px',
+                },
+                input: {
+                  position: 'relative',
+                  color: 'grey.400',
+                  fontFamily: 'var(--font-primary)',
+                  lineHeight: 1,
+                  fontSize: '18px',
+                  marginLeft: '45px',
+                  marginTop: '0px',
+                  marginBottom: '10px',
+                  paddingLeft: '0px',
+                  paddingTop: '8px',
+                  paddingRight: '0px',
+                  paddingBottom: '0px',
+                  width: '270px',
+                },
+                label: {
+                  color: 'grey.400',
+                  fontFamily: 'var(--font-primary)',
+                  lineHeight: 1,
+                  fontSize: '18px',
+                  marginLeft: '30px',
+                },
+                p: {
+                  color: 'grey.400',
+                  fontFamily: 'var(--font-primary)',
+                  lineHeight: 1,
+                  display: 'flex',
+                  justifyContent: 'start',
+                  alignItems: 'start',
+                  paddingLeft: '0px',
+                },
+                legend: {
+                  color: 'grey.400',
+                  fontFamily: 'var(--font-primary)',
+                  lineHeight: 1,
+                  marginLeft: '30px',
+                  marginTop: '0px',
+                  marginBottom: '0px',
+                  paddingLeft: '0px',
+                  paddingTop: '0px',
+                  paddingBottom: '0px',
+                },
+                span: { color: 'grey.400', fontFamily: 'var(--font-primary)', lineHeight: 1 },
+              }}
+            />
+
+            <EmailIcon
+              sx={{
+                position: 'absolute',
+                fill: 'lightgray',
+                top: '10px',
+                left: '10px',
+              }}
+            />
+          </div>
+          <div className={css.container_input}>
+            <TextField
+              name="password"
+              type="password"
+              label="Could change password"
```

Comment:
Dałabym po prostu 'New password'

Reply:
Tu mam wątpliwości, jak powiadomić Usera, że nie musi (a może) zmieniać hasła. Możliwe jest uaktualnienie first name, i/albo currency... oraz wszystkiego...

Reply:
No to może 'Change your password'
Albo "Would you like to change your password?" (pewnie za długo)

Reply:
I jeszcze "New password (optional)" mi przyszło do głowy